fabric-events-lifecycle-v0/src/main/resources/fabric-events-lifecycle-v0.mixins.json

```diff
@@ -8,7 +8,8 @@
   ],
   "client": [
     "MixinItemStack",
-    "MixinMinecraftClient"
+    "MixinMinecraftClient",
+    "MixinInGameHud"
```

Comment:
Alphabetical order on mixins - this should go at the top of the `client` list, not the bottom

Reply:
Done.